### DISCUSSION TITLE
Isaac Lab | Fails to run tutorials\00_sim\create_empty.py | conflicting gpu error

### DISCUSSION THREAD

**Discussion by sosno** [isaac-sim/IsaacLab#1588] [General]
I am trying to get IsaacLab running on my system for the first time by following the binary installation instructions. I’m following the tutorial found at:  https://isaac-sim.github.io/IsaacLab/main/source/setup/installation/binaries_installation.html

**System Info:**
Isaac Sim Version:   4.2.0
Isaac Lab Version:  1.2
Operating System:  Windows 11
GPU:  RTX 4090 Laptop
Driver Version:  Studio Driver -566.14

**Detailed Description**
I installed Isaac Lab and was on the last step “Verifying the Isaac Lab installation” where we are asked to run “isaaclab.bat -p source\standalone\tutorials\00_sim\create_empty.py”. When I attempt to run this file I get an error pop up menu that says “GPU Foundation” “Failed to create any GPU devices, including an attempt with compatibility mode. If I look in the terminal window I can see an error that says [Error] [gpu.foundation.plugin] Multiple Installable Client Drivers are found for the same GPU on the system.”

Note that I can run Isaac Sim 4.2 and USD Composer just fine. If I run “isaaclab.bat -s” it starts up just fine.

I’ve seen some posts on the internet with a similar error related to Isaac Sim and in those posts it was suggested to revert to an older driver. But I can’t imagine that is my issue since Isaac Sim runs just fine from the launcher. Also, I’ve tried two drivers now.

**Steps to Reproduce**
1. Installed Isaac Sim 4.2 via Launcher
2. Cloned the Isaac Lab repo
3. I created the symbolic link etc.
4. Installed isaac lab using “isaaclab.bat --install”
5. Tried running: isaaclab.bat -p source\standalone\tutorials\00_sim\create_empty.py
![image](https://github.com/user-attachments/assets/774e6345-f048-4f95-ad0a-6c383183d210)

**What I’ve Tried**
Note that I can run Isaac Sim 4.2 and USD Composer just fine.

If I run “isaaclab.bat -s” it starts up just fine.

I added some print statements to the create_empty.py code and I can see that the last line that is called is this: app_launcher = AppLauncher(args_cli).

After reading online I tried updating my NVIDIA gpu driver which is what I currently have listed in this description. So two drivers were attempted and encountered the same issue.

**Additional Context**
I saw some verbiage about Cuda 11 and 12 and using the correct drivers. I tried to see if I have CUDA installed on my system and couldn’t really tell. I tried running nvcc --version but the command nvcc doesn’t work.

Here’s another piece of info that may be relevant. When I run the command “Isaaclab.bat -s” and it successfully launches isaac sim, I noticed the warnings below in the terminal. This is telling me that for some reason when IsaacSim is run, it must be somehow able to disable the redundant unsupported non-Nvidia drivers, but maybe the same is not happening when I run the isaacLab tutorial files.

![image](https://github.com/user-attachments/assets/b893f157-a43f-40e8-8c2b-b6ea32f13dde)

**Comment by lfcars** [comment]
I experienced a similar problem. Initially, I suspected the issue was related to a Windows update and tried installing multiple previous versions of Nvidia's drivers. However, those efforts, along with other tests, didn't resolve the problem.

It was finally resolved when I disabled my laptop's second display adapter (Intel(R) UHD Graphics) in the device manager.

**Reply by shaoxiang** [reply]
I also used the same solution to disable Intel's graphics card to solve this problem, but this made my computer very slow.

**Reply by sosno** [reply]
Thanks, I will need to try this.

**Reply by shaoxiang** [reply]
I was still selecting discrete graphics in the BIOS, and now there is no longer this issue. The computer runs very smoothly.

**Comment by RandomOakForest** [comment]
Thanks for posting this. Note that if you have _Windows with CUDA 12, the GPU driver version 552.86 is required._

**Reply by sosno** [reply]
Thanks for the reply.  So I understand correctly...  Isaac Lab requires a specific version of driver?...  even though Isaac Sim works just fine?  I'm running driver version 566.14 right now and that works fine with Isaac Sim.  Usually NVIDIA drivers are backward compatible.

**Comment by 23and1** [comment]
Thanks @lfcars!! I've been having the same issue for a week now. I tried disabling the Intel UHD Graphics in the device manager and it worked! If there is a way to run it without having to disable that, I would be interested in that as well.

**Comment by RandomOakForest** [comment]
Thank you for following up. This seems to be a platform specific issue, related to iGPU and dGPU drivers and the BIOS. Probably the next step would be to try the latest release of Isaac Lab and see if this problem persists.

**Reply by ts-jls** [reply]
I noticed the same problem happens with the latest Isaac Sim v4.5.  I'm able to get it to run following the instructions of others, above.  But it's not ideal to turn off my Intel graphics card because it homes with a host of other issues.  When I look at the terminal when I launch Isaac Sim, I can see in previous versions it would simply issue a warning but it would still run.  So previous versions must of had a fall back for scenarios like this, and it seems that was removed or changed in the latest version of Isaac Sim and Lab.

**Comment by tovo99** [comment]
If you don't want to disable the display adapter, you can set the VK_ICD_FILENAMES variable before starting isaacsim. Here is a sample .bat file to start isaacsim without the error: 

set VK_ICD_FILENAMES=_**C:\Windows\....\nv-vk64.json**_
echo Starting Isaac Sim...
call isaac-sim.bat

Replace _**C:\Windows\....\nv-vk64.json**_ with your path to nv-vk64.json. (You can just search in C:\Windows)
